hadoop/Dockerfile

```diff
@@ -8,12 +8,12 @@ ARG JMX_EXPORTER
 ARG PROTOBUF
 ARG TARGETARCH
 ARG TARGETOS
+ARG STACKABLE_USER_UID
```

Comment:
Just a thought, should these be defaulting to something so that they don't need specifying when doing a local build?

Nothing to change right now, just wanted to ask.

Reply:
They should default to the current `1000`

See here: https://github.com/stackabletech/docker-images/pull/849/files#diff-6e03d3d89630aba3403afea3514a3ac5948e70d1ced091012e56446f614da340

Or do you mean something different?

Reply:
I see that I also put it in the PR description actually. I believe that works if you use image_tools 0.0.13

> Using the new https://github.com/stackabletech/image-tools/pull/41 to support global arguments this extracts the user id, user name and gid into arguments that can be changed easily.
They still stay at the 1000 we use so far even though that is not optimal and needs to be changed as well.

Reply:
> They should default to the current 1000

The default to `1000` in bake, but not in the Dockerfile (the ARG has no default).

Reply:
That is by intention, yes. The whole idea here was to only specify the UID in a single place so we can easily update it once we are ready to do so.

Reply:
I'm meaning the Dockerfile should have defaults (bake overrides those defaults), but since we don't do that for any other arg, we can ignore it.